src/PeopleSDKAdapter.test.js

```diff
@@ -119,6 +121,54 @@ describe('People SDK Adapter', () => {
     });
   });
 
+  describe('searchPeople', () => {
+    test('returns an observable', () => {
+      expect(isObservable(peopleSDKAdapter.searchPeople(query))).toBeTruthy();
+    });
+
+    test('emits persons list on subscription', (done) => {
+      mockSDK.people.list = jest.fn(() => Promise.resolve({
+        items: [
+          {
+            id: 'personIDCurrentUser',
+            emails: ['email@cisco.com'],
+            displayName: 'Webex Components',
+            firstName: 'Webex',
+            lastName: 'Components',
+            avatar: 'avatar',
+            orgId: 'orgID',
+          },
+        ],
+      }));
+
+      peopleSDKAdapter.searchPeople(query).subscribe((peopleList) => {
+        expect(peopleList).toEqual([{
+          id: 'personIDCurrentUser',
```

Comment:
here, ID should be uppercase, because this is a Person object as defined here:

https://github.com/webex/component-adapter-interfaces/blob/master/src/PeopleAdapter.js#L6

Reply:
changes done